parm/sfs/postcntrl_sfs.xml

```diff
@@ -299,6 +299,12 @@
        <table_info>NCEP</table_info>
        <scale>3.0</scale>
     </param>
+    
+    <param>
```

Comment:
@NeilBarton-NOAA SSRUN is already in the SFS UPP control file. Please confirm that atmosphere top downward shortwave is needed.

Reply:
yes, I was confused with the gefs xml. AVE_DSWRF_ON_TOP_OF_ATMOS is the added value